ambry-api/src/main/java/com/github/ambry/config/MySqlNamedBlobDbConfig.java

```diff
@@ -52,6 +54,22 @@ public class MySqlNamedBlobDbConfig {
   @Default("100")
   public final int listMaxResults;
 
+  /**
+   * The maximum number of entries to return per each stale blobs pull request.
+   */
+  @Config(STALE_MAX_RESULTS)
+  @Default("1000")
+  public final int staleMaxResults;
+
+  /**
+   * The maximum number of days for a stale blob to say uncleaned.
+   */
+  @Config(STALE_MAX_DAYS)
+  @Default("5")
+  public final int staleMaxDays;
```

Comment:
nit: Honestly I think the variable names and the config names should be more descriptive. Something like
`query.stale.data.max.results` and `stale.data.retention.day`

Reply:
Done